package.json

```diff
@@ -14,6 +15,8 @@
     "dotenv": "^6.0.0",
     "ejs": "^2.6.1",
     "errorhandler": "^1.5.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-plugin-react": "^7.16.0",
```

Comment:
Nice work Ivy, but I don't understand why we need eslint-plugin-react

Reply:
I figured we'd better have it now since we will be using react for the frontend?

Reply:
Kindly resolve this comments @nziokaivy

Reply:
@nziokaivy  but the front end will be on another repo.